addons/docs/src/frameworks/svelte/extractArgTypes.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable new-cap */
```

Comment:
disable at line level?

Reply:
i disabled the linting rule at line level